packages/web/src/scss/foundation/_links.scss

```diff
@@ -7,11 +8,17 @@
 a {
     color: tokens.$action-link-primary-default;
 
-    &:hover {
-        color: tokens.$action-link-primary-hover;
-    }
-
     &:active {
```

Comment:
I think we can move this code block down to the media query.

https://developer.mozilla.org/en-US/docs/Web/CSS/:active

Reply:
I am not sure if you mean below the media, but if so, I 100 % agree. I just had to fix this in the Pagination :) the active state was first and it was overwritten by the hover state.
it should be:
```
a {
 ...
 @media (hover) {
  &:hover {}
 }
 
 &:active {}
}
```

Reply:
No, I meant inside the code block, because pseudo-class `:active` _typically starts when the user presses down the primary mouse button_.

Anyway good point with overwriting!

Reply:
Element can be activated even by touch, so I don't think it should be in the hover MQ. https://html.spec.whatwg.org/multipage/semantics-other.html#selector-active

Reply:
As we in Czechia say: „Vyprovodím se sama...“. 🚪

Reply:
Thank you, I'm closing this conversation.